README.md

```diff
@@ -0,0 +1,16 @@
+# Zoom Telegram Reminder Bot
+
+This bot notifies a Telegram channel when a new lecture is about to start. Members of the channel can easily register new lectures and view the existing ones. The project has been created primarily for our student Telegram group, but you can use it as well. 
```

Comment:
I would leave out "but you can use it as well". This is implied by the project being open source.